mkdocs.yml

```diff
@@ -16,33 +16,33 @@ theme:
 # Pages
 nav:
   - Welcome: "index.md"
-  - Introduction: "01_introduction.md"
-  - Project structure: "20_project_structure.md"
-  - Schema: "30_schema.md"
+  - Introduction: "introduction.md"
+  - Project structure: "project-structure.md"
+  - Schema: "schema.md"
   - User Guide:
-      - Create a research protocol: "41_create_new_protocol.md"
-      - Adopt assessments from the library: "42_adopt_assessments.md"
-      - Create new assessments for a protocol: "43_create_new_assess.md"
-      - Add a feedback section: "44_setup_feedback.md"
-      - Finalize the protocol: "45_finalize_protocol.md"
-      - Toolkit: "46_tools.md"
+      - Create a research protocol: "user_guide/create-new-protocol.md"
+      - Adopt assessments from the library: "user_guide/adopt-assessments.md"
+      - Create new assessments for a protocol: "user_guide/create-new-assess.md"
+      - Add a feedback section: "user_guide/setup-feedback.md"
+      - Finalize the protocol: "user_guide/finalize-protocol.md"
+      - Toolkit: "user_guide/tools.md"
   - Tutorial:
-      - Intro: "50_using_reproschema.md"
-      - Create a research protocol: "51_create_new_protocol.md"
-      - Creating a new activity: "52_create_new_activity.md"
-      - Tips to make your life easier: "53_tips_to_make_your_life_easier.md"
-      - Translate a questionnaire: "54_translating_an_activity.md"
-      - Demographic information : "55_collecting_demographics_information.md"
+      - Intro: "tutorials/using_reproschema.md"
+      - Create a research protocol: "tutorials/create-new_protocol.md"
```

Comment:
here, change it to `create-new-protocol.md`?

Reply:
done